symfony/plugins/orangehrmHelpPlugin/lib/processor/ZendeskHelpProcessor.php

```diff
@@ -22,6 +22,7 @@ class ZendeskHelpProcessor implements HelpProcessor {
     const DEFAULT_CONTENT_TYPE = "application/json";
     const ZENDESK_SEARCH_DIRECTORY_URL = '/api/v2/help_center/articles/search.json?';
```

Comment:
Is this `.json` required. Also, you can move ? to particular place, rather than here

Reply:
removed the DIRECTORY part of the variable